lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -268,9 +273,16 @@ class ReportUIFeatures {
         filterInput.checked = thirdPartyRows.size === urlItems.length;
       }
 
-      // Finally, add checkbox to the DOM.
+      // Add checkbox to the DOM.
       if (!tableEl.parentNode) return; // Keep tsc happy.
       tableEl.parentNode.insertBefore(filterTemplate, tableEl);
+
+      // Hide third-party rows for some audits by default.
```

Comment:
Why aren't we just hiding these at the audit level?

If it's just for JSON usage I'd prefer personally prefer to stick that into extendedInfo or debugdata or something. The contract we'd extended to consumers of the LHR so far is that items in `.details` are meant to be rendered.

Reply:
yea you're right. This isn't normally possible, b/c we disable the checkbox by default if there are only all of 1p/3p.

Reply:
> The contract we'd extended to consumers of the LHR so far is that items in .details are meant to be rendered.

Sorry. I failed to address this.

`extendedInfo` doesn't persist to the LHR, so I don't like that.

How would I add a debugdata in addition to the table details?

Can this be done in a follow up (and be a blocker for becoming enabled by default)?

Reply:
Yeah sure, not a blocker for now